config/fastly/snippets/bad_bot_detection.vcl

```diff
@@ -0,0 +1,140 @@
+sub vcl_recv {
+  # Disallow bot traffic based on user-agent
+  if (
+    req.http.user-agent ~ "^$"
+    || req.http.user-agent ~ "^Java"
+    || req.http.user-agent ~ "^Jakarta"
+    || req.http.user-agent ~ "IDBot"
+    || req.http.user-agent ~ "id-search"
+    || req.http.user-agent ~ "User-Agent"
+    || req.http.user-agent ~ "compatible ;"
+    || req.http.user-agent ~ "ConveraCrawler"
+    || req.http.user-agent ~ "^Mozilla$"
+    || req.http.user-agent ~ "libwww"
+    || req.http.user-agent ~ "lwp-trivial"
+    || req.http.user-agent ~ "PHP/"
```

Comment:
Does this mean any script created written with PHP would be considered a bot?

Reply:
Most likely - I can remove this one.